config/locales/en.yml

```diff
@@ -1564,6 +1565,7 @@ en:
         message_failed: "Failed to import %{nr} protocol/s."
         message_ok: "Successfully imported %{nr} protocol/s."
         message_ok_pio: "Successfully imported protocol from protocols.io file."
+        message_warn_truncated: "Some fields have been shortened due to being too long."
```

Comment:
Text should be different. Change it to: "Successfully imported protocol from protocols.io file. However, text in some fields was too long so we had to cut it off."